Dockerfile

```diff
@@ -445,6 +445,7 @@ RUN pip install flashtext && \
     pip install pydub && \
     pip install pydegensac && \
     pip install pytorch-lightning && \
+    pip install https://storage.googleapis.com/tpu-pytorch/wheels/torch_xla-1.6-cp37-cp37m-linux_x86_64.whl && \
```

Comment:
nit: This is not in pypi yet, right? Shall we add a comment here to later use pypi?